Dockerfile

```
@@ -10,24 +10,21 @@ LABEL org.opencontainers.image.source "https://github.com/thelounge/thelounge-do
 LABEL org.opencontainers.image.version "${THELOUNGE_VERSION}"
 LABEL org.opencontainers.image.licenses "MIT"
 
-ENV NODE_ENV production
-
+EXPOSE 9000
 ENV THELOUNGE_HOME "/var/opt/thelounge"
-VOLUME "${THELOUNGE_HOME}"
-
-# Expose HTTP.
-ENV PORT 9000
-EXPOSE ${PORT}
-
-ENTRYPOINT ["docker-entrypoint.sh"]
-CMD ["thelounge", "start"]
 
-COPY docker-entrypoint.sh /usr/local/bin/docker-entrypoint.sh
-
-# Install thelounge.
+ENV NODE_ENV production
 RUN apk --update --no-cache --virtual build-deps add python3 build-base git && \
     ln -sf python3 /usr/bin/python && \
     yarn --non-interactive --frozen-lockfile global add thelounge@${THELOUNGE_VERSION} && \
     yarn --non-interactive cache clean && \
     apk del --purge build-deps && \
     rm -rf /root/.cache /tmp /usr/bin/python
+
+RUN install -d -o node -g node "${THELOUNGE_HOME}"
```

Might as well merge this with the previous RUN layer, IMO.

```suggestion
    rm -rf /root/.cache /tmp /usr/bin/python && \
    install -d -o node -g node "${THELOUNGE_HOME}"
```

---

why? It's less readable and there's no point
There's no problem with adding an additional layer

---

less bits more better, idk - it doesn't change anything functionally tho

---

why?

It's all the same for a user, it doesn't matter.
We aren't remotely impacting run times nor performance.

Readability and maintainability of the dockerfile is more important.

Each run step should do one logical thing, the run step above does installation of all software packages.

Adding a directory for TL runtime doesn't fall under that imo

---

Yeah I think they should be separate. The only reason to join RUN statements is if it results in a smaller layer footprint, imo.